### DISCUSSION TITLE
Modify ManageRelatedRecords form actions

### DISCUSSION THREAD

**Discussion by fginer** [filamentphp/filament#12872] [Help]
### Package

Panel builder

### Package Version

v3.2.0

### How can we help you?

Hi everyone.

In a CreateRecord file you can modify form actions using getFormActions method, but in a ManageRelatedRecords I don't know how to do that. Is it possible?

Thanks in advance.

**Comment by fginer** [comment]
I want to disable "create and create another" button.

**Reply by leandrocfe** [reply]
Why not hide the CreateAction?

**Reply by fginer** [reply]
I want to create elements from ManageRelatedRecords inheritance class, but I don't want the "create and create another" button.

**Comment by Abdelhalim-Merabet** [comment]
``
Tables\Actions\CreateAction::make()
                    ->createAnother(false)
``

have you tried this ?
